modules/running-compliance-scans.adoc

```diff
@@ -13,6 +13,11 @@ You can run a scan using the Center for Internet Security (CIS) profiles. For co
 For all-in-one control plane and worker nodes, the compliance scan runs twice on the worker and control plane nodes. The compliance scan might generate inconsistent scan results. You can avoid inconsistent results by defining only a single role in the `ScanSetting` object.
 ====
 
+[IMPORTANT]
+====
+Compliance Operator scans will report INCONSISTENT on clusters with multi-architecture compute machines whether the control plane uses `aarch64` or `x86` CPUs.
```

Comment:
Minor nit: We could use Compliance Check Results or Compliance Scans with `INCONSISTENT` results to be more descriptive (also using a literal for a technical term?).

Not sure how important this is, but we could also say something to the effect of:

> This is due to the same rule behaving differently on different architectures. This should only be applicable for node scans, where the Compliance Operator aggregates results from multiple nodes into a single result.